en/user-guide/history/history.rst

```diff
@@ -25,15 +25,12 @@ The "logged events" window is composed of 4 parts.
 Replay function
 ---------------
 
-.. note::
-  Doesn't work yet for QGEP.
+If you have configured the replay option, you can replay actions. Example below:
```

Comment:
Can you make "configured the replay option" a hyperlink?

Reply:
@m-kuhn I don't understand what you mean by hyperlink, but if @lbartoletti does, please make the change.
LGTM otherwise

Reply:
@ponceta if someone reads this and has not configured the replay option, we should be helpful enough to show him where he can find the information on how to do that.

Reply:
@m-kuhn the first screen shows how to enable the replay function

Reply:
https://qgep.github.io/docs/fr/_images/pg_hv_configuration.png

Reply:
I see. 
Can you add a link to the "Plugin configuration" section? And/or write "enabled the replay option in the plugin configuration"?
Maybe it's just me as an outsider who is not yet completely familiar with pg history viewer, but I think it would be a bit clearer in terms of taxonomy.